src/adm/settings.rs

```diff
@@ -316,6 +400,13 @@ impl From<&mut Settings> for HandlerResult<AdmFilter> {
             ignore_list,
             all_include_regions,
             legacy_list,
+            last_updated: match &source.starts_with("gs://") {
+                true => Some(chrono::Utc::now()),
+                false => None,
+            },
```

Comment:
They added this in Rust 1.50!
```suggestion
            last_updated: source.starts_with("gs://").then(|| chrono::Utc::now()),
```